package.json

```diff
@@ -3,20 +3,26 @@
   "version": "5.0.2",
   "description": "React components for Heroku's Malibu SVG icon system",
   "main": "lib/index.js",
-  "module": "es/index.js",
+  "module": "lib/index.mjs",
```

Comment:
Can you double check this change? I don't know enough off the top of my head.

Reply:
As far as I can tell it's to help node distinguish between common js and es module.